src/bootstrap/src/core/build_steps/dist.rs

```diff
@@ -462,8 +462,11 @@ impl Step for Rustc {
                 let src_dir = builder.sysroot_libdir(compiler, host).parent().unwrap().join("bin");
                 let rust_lld = exe("rust-lld", compiler.host);
                 builder.copy_link(&src_dir.join(&rust_lld), &dst_dir.join(&rust_lld));
-                let self_contained_lld_src_dir = src_dir.join("gcc-ld");
-                let self_contained_lld_dst_dir = dst_dir.join("gcc-ld");
+
+                let gcc_ld = exe("gcc-ld", compiler.host);
```

Comment:
Is this exe call about a gcc-ld executable ? It’s a directory where the lld binaries are.

Reply:
Right, my bad. Thanks!